src/views/admin/Grants/createGrant.tsx

```diff
@@ -93,7 +93,6 @@ const CreateGrant = ({ initialValues }: CREATE_GRANT_TYPES) => {
                   options={TYPE_OPTIONS}
                   errorSelect={errors.type}
                 />
-                {errors.type && touched.type && errors.type}
```

Comment:
why you have removed this line
{errors.type && touched.type && errors.type}

Reply:
beacuase i am getting result without it